src/custom/components/Header/URLWarning/index.tsx

```diff
@@ -89,7 +90,8 @@ export default function URLWarning() {
   const { chainId = ChainId.MAINNET } = useWalletInfo()
 
   // Ger announcement if there's one
-  const announcementText = useGetAnnouncement(chainId)
+  const { announcementBanner } = useFlags()
+  const announcementText = useGetAnnouncement(chainId) || announcementBanner
```

Comment:
This flag doesn't actually exist, I used it for testing and was an artifact from Anuj's hackaton project.
I'll remove it from this PR.

We should get back to this with more time and overhaul how the banner text is loaded using launch darkly, if we decide to keep it